src/hooks/useYoutubeApi.test.jsx

```diff
@@ -0,0 +1,52 @@
+import { act, renderHook } from '@testing-library/react-hooks';
```

Comment:
It seems this test doesn't run when executing `npm test`:

![image](https://user-images.githubusercontent.com/124746/115485073-9d304300-a219-11eb-9112-59065aed5281.png)